x-pack/test/functional/apps/infra/hosts_view.ts

```diff
@@ -209,16 +220,37 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
             expect(hosts.length).to.equal(9);
           });
 
-          it('should show all section as collapsable', async () => {
+          it('should show all section as collapsible', async () => {
             await pageObjects.assetDetails.metadataSectionCollapsibleExist();
             await pageObjects.assetDetails.alertsSectionCollapsibleExist();
             await pageObjects.assetDetails.metricsSectionCollapsibleExist();
+            await pageObjects.assetDetails.servicesSectionCollapsibleExist();
           });
 
           it('should show alerts', async () => {
             await pageObjects.header.waitUntilLoadingHasFinished();
             await pageObjects.assetDetails.overviewAlertsTitleExists();
           });
+
```

Comment:
In hindsight I wish i'd created fresh host data instead of trying to keep this in the Overview tab describe.  Creating new host data would have required me to update other tests.  I'd like to go back and use the infra synthtrace client instead of the host data in esArchiver files to make it easier to update and create the test data.